docs/source/nlp/megatron_finetuning.rst

```diff
@@ -0,0 +1,57 @@
+.. _megatron_finetuning:
+
+Megatron-LM for Downstream Tasks
+================================
+
+Megatron-LM :cite:`nlp-megatron-shoeybi2019megatron` is a large, powerful transformer developed by the Applied Deep Learning Research team at NVIDIA.
```

Comment:
Add "It is an improved version of BERT which is trained with masked language modelling (MLM) approach?